### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# First Release
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-1.0.md:1]: Can you put the release version there?

Do you want the first release to be 1.0? or 0.1?

**Comment by msau42** [comment]
Can you also update the README to document the binary flags, and also have the support matrix like in the other sidecars?

File: CHANGELOG/CHANGELOG-1.0.md (hunk 1 above)
Comment (xing-yang) [reply CHANGELOG/CHANGELOG-1.0.md:1]: We should probably do a 0.1 to start with.

**Comment by xing-yang** [comment]
Will submit another PR to update README to document the flags and support matrix.

**Comment by xing-yang** [comment]
@msau42 addressed your comments.  PTAL.

**Pull request merged**